assertsprocessor/sampler.go

```diff
@@ -191,6 +191,11 @@ func (s *sampler) updateTrace(namespace string, service string, ts *traceSegment
 	}
 }
 
+func (s *sampler) ignoreErrorType(span *ptrace.Span) bool {
```

Comment:
Should we ignore client errors for both `inbound` and `outbound` or only for `inbound`? `outbound`  client errors are an indication of a problem in the calling component.